public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -410,6 +408,10 @@ export class LokiDatasource
         expression = addParserToQuery(expression, 'json');
         break;
       }
+      case 'ADD_NO_PIPELINE_ERROR': {
```

Comment:
I guess at some point we should move these static strings to an Enum. But nothing to do right now.